onnxruntime/contrib_ops/cuda/bert/skip_layer_norm_impl.cu

```diff
@@ -106,13 +172,13 @@ bool LaunchSkipLayerNormKernel(
     const void* skip,
```

Comment:
Can we delete this LaunchSkipLayerNormKernel function, and rename ComputeSkipLayerNorm to LaunchSkipLayerNormKernel? The code is something like this:

LaunchSkipLayerNormKernel <T>(
        stream,
        reinterpret_cast<T*>(output),
        ...
        T(epsilon),
        ...
        );